ansible/playbook/tasks/update.yml

```diff
@@ -19,6 +19,64 @@
       - percona-clickhouse-server
       - pmm2-client
       - dbaas-controller
+    pmm_rendering_dependencies:
```

Comment:
Do we really have to specify them all, or we can specify only core dependencies, and everything else will be installed as a dependency?

Reply:
https://grafana.com/docs/grafana/latest/administration/image_rendering/#grafana-image-renderer-plugin-and-remote-rendering-service

![Screenshot_20200902_124259](https://user-images.githubusercontent.com/25184101/91966185-2ee65100-ed1a-11ea-89ab-be027d1aea1d.png)

Reply:
That link should be in the comment then